src/packages/@ncigdc/utils/withRouter.js

```diff
@@ -57,7 +68,3 @@ const enhance = compose(
     },
   }))
 );
-
-const withBetterRouter = Wrapped => enhance(props => <Wrapped {...props} />);
```

Comment:
Since in SsmsTable that's just local name for the imported default from `withRouther.js`, as long as a default is exported from here, the other one shouldn't care, right?

As for the removed const, turns out that wrapper is implicit in Recompose's `compose` method.